### PR TITLE
Amy/visual bugs

### PR DESCRIPTION
**Pull request by amycshuang**
## Overview

Fixed visual bugs found using testFlight.

## Changes Made

Changed constraints for visual bugs for text getting cut off on some screens. 

## Test Coverage

Manual testing with simulator.

## Screenshots

<details>

  <summary></summary>

https://user-images.githubusercontent.com/58535892/105875640-a9628680-5fc3-11eb-8d32-f6999df65be4.mp4

bug2
bug3

https://user-images.githubusercontent.com/58535892/105875700-ba12fc80-5fc3-11eb-9963-46ec8b4701f0.mp4

bug6

  
  

</details>

### CODE REVIEW

File: CoffeeChat/Controller/HomeViewController.swift [hunk 1]
```diff
@@ -143,7 +143,7 @@ class HomeViewController: UIViewController {
         menu.leftSide = true
         menu.statusBarEndAlpha = 0
         menu.menuWidth = view.frame.width * 0.8
-        present(menu, animated: true)
+        present(menu, animated: true, completion: nil)
```
[the remark this answers is not in the document]
Comment (amycshuang) [reply CoffeeChat/Controller/HomeViewController.swift:146]: Oh I thought that might have been the reason for the black flickering when opening the side menu options, but I forgot to remove it haha.

File: CoffeeChat.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1045,7 +1045,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = ZGMCXU7X3U;
 				INFOPLIST_FILE = CoffeeChat/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```
Comment (lucyxubroad) [inline CoffeeChat.xcodeproj/project.pbxproj:1048]: do we want 14? I think decided 13 is good

**Review by lucyxubroad — APPROVED** [review]
great work!!! :)

File: CoffeeChat.xcodeproj/project.pbxproj (hunk 2 above)
Comment (amycshuang) [reply CoffeeChat.xcodeproj/project.pbxproj:1048]: It won't let me run the simulator if I use 13. It says "Compiling for iOS 13.0, but module 'FutureNova' has a minimum deployment target of iOS 14.0" ?

**Pull request merged**